.github/workflows/pyopenms-wheels.yml

```diff
@@ -181,6 +181,11 @@ jobs:
         export CFLAGS="-Xpreprocessor -fopenmp $CFLAGS"
         export CXXFLAGS="-Xpreprocessor -fopenmp $CXXFLAGS"
 
+        # Need to explicitly point to the icu4c lib installation see: https://web.archive.org/web/20231126001229/https://stackoverflow.com/questions/52510499/the-following-icu-libraries-were-not-found-i18n-required
```

Comment:
Hmm maybe either add to CMake, or just build with a shared lib boost by using BOOST_USE_STATIC=OFF 5 lines later.

See also:
https://github.com/OpenMS/OpenMS/blob/c112411600c4a623d7a01f15c206ff827ba44401/cmake/cmake_findExternalLibs.cmake#L63

Reply:
It's mostly an upstream issue since they do not specify their transitive dependencies in the CMake config files correctly. But nothing we can do about it.